packages/harmony/src/components/button/FilterButton/FilterButton.tsx

```diff
@@ -218,20 +218,21 @@ export const FilterButton = forwardRef(function FilterButton<
     >
       {(activeValue) =>
         filteredOptions.map((option, index) => (
-          <FilterButtonOption
+          <MenuItem
+            variant='option'
             ref={(el) => {
               if (optionRefs && optionRefs.current && el) {
                 optionRefs.current[index] = el
               }
             }}
             key={option.value}
-            option={option}
+            {...option}
             onChange={handleChange}
-            activeValue={activeValue}
+            isActive={option.value === activeValue}
```

Comment:
Just confirming: option.value should always be unique, right?
Otherwise this could light up two things at once